common/web/keyboard-processor/tests/cases/basic-engine.js

```diff
@@ -1,20 +1,18 @@
-var assert = require('chai').assert;
-let fs = require('fs');
-let vm = require('vm');
+import { assert } from 'chai';
+import fs from 'fs';
+import vm from 'vm';
 
-let KeyboardProcessor = require('../../build/index.bundled.js');
-
-// Required initialization setup.
-global.com = KeyboardProcessor.com; // exports all keyboard-processor namespacing.
-
-let KMWRecorder = require('../../../recorder/build/nodeProctor');
+import KeyboardProcessor from '@keymanapp/keyboard-processor/build/obj/text/keyboardProcessor.js';
+import { KeyboardTest } from '@keymanapp/recorder-core/build/obj/index.js';
+import NodeProctor from '@keymanapp/recorder-core/build/obj/nodeProctor.js';
```

Comment:
It'd be good to be able to reduce this to:
```suggestion
import KeyboardProcessor from '@keymanapp/keyboard-processor';
import NodeProctor, { KeyboardTest } from '@keymanapp/recorder-core';
```